docs/src/terminology.md

```diff
@@ -280,6 +280,10 @@ A scarce, fungible member of a set of tokens.
 
 One or more [instructions](terminology.md#instruction) signed by the [client](terminology.md#client) and executed atomically.
 
+## transaction confirmations
+
+The number of blocks since a transaction's block (inclusive) that have a supermajority of ledger votes. A transaction is fully confirmed when its block becomes a [root](terminology.md#root).
```

Comment:
The transaction is "finalized", not "fully confirmed" when its block becomes a root.  Also, that first sentence can use some work.  How about: "The number of confirmed blocks since the transaction was accepted onto the ledger." Might need a definition for "confirmed block" too, and if so, that can be used to simplify the definition of "confirmation time".

Reply:
Updated and added terms for:
* "confirmed block"
* "supermajority"